demos/continuous_batching/README.md

```diff
@@ -1,4 +1,15 @@
-# How to serve LLM models with Continous Batching via OpenAI API {#ovms_demos_continous_batching}
+# How to serve LLM models with Continous Batching via OpenAI API {#ovms_demos_continuous_batching}
+
+
+## Build the docker image
+
+To use the latest features of the model server build the image from the main branch.
+Public image will support this demo starting from version 2024.2.
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server
+make release_image
+```
```

Comment:
can we change command: PIP_EXTRA_INDEX_URL="https://download.pytorch.org/whl/cpu" python3 -m pip install ...

into two-liner:
export PIP_EXTRA_INDEX_URL="https://download.pytorch.org/whl/cpu"
python3 -m pip install "optimum-intel[nncf,openvino]"@git+https://github.com/huggingface/optimum-intel.git openvino-tokenizers

?  It will help with automated testing